examples/cosmos-sdk/transfer/transfer.md

```diff
@@ -110,171 +90,152 @@ View ==
+For the moment, `reactor.py` is a stub, which will for every step of the test scenario simply print the information about the action.
+We will see how to extend the stub in section [Expanding reactors](#expanding-reactors).
 
-Now we can update the chain parameters.
+## Setting up chains
 
-<!-- $MDX dir=transfer -->
-```sh
-$ atomkraft chain config chain_id test-sdk
-$ atomkraft chain config binary ./cosmos-sdk/build/simd
-$ atomkraft chain config prefix cosmos
+Our ultimate goal is to execute tests on a chain.
+Now we set up the testnet to be used with Atomkraft.
+
+If you were following along the instructions in [INSTALLATION.md](/INSTALLATION.md), you are already have `gaiad` installed.
+Since `gaiad` is a default testnet in Atomkraft, there is nothing else to be done.
+
+Working with other chains is possible too, by installing them and then invoking:
+
+```
+atomkraft chain config chain_id <name_of_the_chain>
+atomkraft chain config binary <path-to-the-chain-binary>
+atomkraft chain config prefix <prefix?>
 ```
 
-### Executing tests
+## Executing tests
 
-We have some traces and a reactor stub ready. Now we can smoke-test the test.
+All three elements for running a test are now in place: a testnet, a test scenario, and a reactor.
+Thus, we are ready to run some tests (though, trivial ones since the reactor is still only a stub).
 
-<!-- $MDX dir=transfer -->
+<!-- $MDX dir=transfer
 ```sh
 $ poetry run atomkraft test trace --trace traces/violation1.itf.json --reactor reactors/reactor.py --keypath action.tag
 ...
 Successfully executed trace traces/violation1.itf.json
 ...
 ```
+-->
 
-For now, this just prints the name of the action tag. Let's complete the reactor stub.
-
-### Updating reactors
-
-Update `reactors/reactor.py` with the following complete reactor code.
-
-<!-- $MDX dir=transfer -->
-```sh
-$ curl -Lo reactors/reactor.py https://raw.githubusercontent.com/informalsystems/atomkraft/dev/examples/cosmos-sdk/transfer/reactor.py
-...
-$ cat reactors/reactor.py
-import time
-
-from modelator.pytest.decorators import step
-from terra_sdk.client.lcd import LCDClient
-from terra_sdk.client.lcd.api.tx import CreateTxOptions
-from terra_sdk.core import Coin, Coins
-from terra_sdk.core.bank import MsgSend
-from terra_sdk.core.fee import Fee
-from terra_sdk.key.mnemonic import MnemonicKey
-
-
-@step("Init")
-def init(testnet, action):
-    print("Step: Init")
-    testnet.n_account = action["value"]["n_wallet"]
-    testnet.verbose = True
-
-    testnet.oneshot()
-    time.sleep(10)
-
-
-@step("Transfer")
-def transfer(testnet, action):
-    print("Step: Transfer")
-
-    rest_endpoint = testnet.get_validator_port(0, "lcd")
-    lcdclient = LCDClient(
-        url=rest_endpoint,
-        chain_id=testnet.chain_id,
-        gas_prices=f"10{testnet.denom}",
-        gas_adjustment=0.1,
-    )
+```
+atomkraft test trace --trace traces/violation1.itf.json --reactor reactors/reactor.py --keypath action.tag
```

Comment:
This will only work if the suggestion from [Modelator issue 252](https://github.com/informalsystems/modelator/issues/252) is accepted.